src/js/messaging/components/compose/MessageSend.jsx

```diff
@@ -39,8 +39,11 @@ class MessageSend extends React.Component {
 }
 
 MessageSend.propTypes = {
+  attachedFiles: React.PropTypes.array,
+  multipleUploads: React.PropTypes.bool,
```

Comment:
Stray propType. Can be fixed when you make the `onAttachmentUpload` required, though.